lib/trailblazer/endpoint/controller.rb

```diff
@@ -121,8 +121,12 @@ def endpoint_config(name, domain_activity: name, **options)
       end
 
       module InstanceMethods
+        # Returns object link between compile-time and run-time config
+        def config_source
+          self.class
+        end
 
-        def endpoint_for(name, config_source: self.class)
```

Comment:
Any specific reason for removing this `config_source` arg? :tulip:

Reply:
yeah, `self.class` needs to be different in case `grape` instance. So I defined a method for `config_source `.

See default method for [rails](https://github.com/trailblazer/trailblazer-endpoint/pull/16/files#diff-311034101006edf191ce47884f27995013a0d742217ce4d3e51018ab0511284aR125) and [grape](
https://github.com/trailblazer/trailblazer-endpoint/pull/16/files#diff-9bf80b8169045c079d120c749ed74cc9f991bc840c3beb65963574138a7b69ceR22)

Reply:
Couldn't you override the entire method instead, for `grape.rb` and change the defaulting there?

Reply:
Hmm, do you mean override `endpoint_for` itself ?

We actually need `config_source` in other methods too (`API.endpoint` and `compile_options_for_controller`).